sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -33,24 +33,24 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
 
   override def render(request: HttpServletRequest): Seq[Node] = {
     val currentTime = System.currentTimeMillis()
-    val content = listener.synchronized {
+    var content : NodeSeq = listener.synchronized {
```

Comment:
I'd rather not switch to a `var` (it's very un-scala), see below for alt suggestion